server/src/main/java/keywhiz/service/daos/GroupDAO.java

```diff
@@ -55,17 +55,17 @@ public void deleteGroup(Group group) {
   public Optional<Group> getGroup(String name) {
     GroupsRecord r = dslContext.fetchOne(GROUPS, GROUPS.NAME.eq(name));
     return Optional.ofNullable(r).map(
-        (rec) -> rec.map(new GroupMapper()));
+        rec -> new GroupMapper().map(rec));
   }
 
   public Optional<Group> getGroupById(long id) {
     GroupsRecord r = dslContext.fetchOne(GROUPS, GROUPS.ID.eq(Math.toIntExact(id)));
     return Optional.ofNullable(r).map(
-        (rec) -> rec.map(new GroupMapper()));
+        rec -> new GroupMapper().map(rec));
   }
 
   public ImmutableSet<Group> getGroups() {
-    List<Group> r = dslContext.select().from(GROUPS).fetch().map(new GroupMapper());
+    List<Group> r = dslContext.selectFrom(GROUPS).fetch().map(new GroupMapper());
```

Comment:
You can't use `.join()` when doing `.selectFrom()` but you can do `.fetchInto()` to get the same result.